components/amount-buttons.jsx

```diff
@@ -70,7 +70,8 @@ var AmountButtons = React.createClass({
     }
     return {
       userInputting: userInputting,
-      valid: true
+      valid: true,
+      errorMessage: ""
```

Comment:
We now display multiple messages depending on the error, so we handle which error it is in the state.